media/css/externalpages/pocket/components/get-inspired.scss

```diff
@@ -0,0 +1,120 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/components/split';
+
+@import '../utils/variables';
+@import '../includes/nav';
+@import '../includes/footer';
+
+$image-path: '/media/img/externalpages/pocket';
+
+.section-heading {
+    @include text-title-sm;
+    font-family: $font-sans-alt;
+    font-weight: 500;
+}
+
+.sub-section-heading {
+    @include text-title-xs;
+    font-family: $font-sans-alt;
+    font-weight: 500;
+}
+
+.section-lede {
+    @include text-body-xl;
+    font-family: $font-sans;
+}
+
+.app-store-badges {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    a {
+        width: 150px;
+        height: 100%;
+
+        &.apple-store {
+            margin-bottom: $spacing-md;
+        }
+
+        img {
+            max-height: 50px;
+            width: 100%;
+        }
+    }
+
+    @media #{$mq-md} {
+        justify-content: flex-start;
+        flex-direction: row;
+
+        .apple-store {
+            margin-bottom: 0 !important;
+            margin-right: $spacing-md;
+        }
+    }
+}
+
+.personal-library {
+    text-align: center;
+    margin: $layout-lg auto;
+
+    .app-store-badges {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+
+        a {
+            width: 210px;
+
+            img {
+                max-height: 100%;
+            }
+        }
+    }
+}
+
+.pocket-premium {
+    margin: $spacing-xl auto;
```

Comment:
```suggestion

```

nit: remove margin here, there's an inner wrapper to manage centering, top margin is provided by the above section and live site leaves the premium section directly against the footer